plugin.yaml

```diff
@@ -1,5 +1,5 @@
 name: "migrate-values"
-version: "0.0.1"
+version: "0.2.2"
```

Comment:
Also updated the version directly here because there won't be a changeset associated with this PR (no changes to the actual migrator functionality itself).